starfish/test/full_pipelines/api/test_iss_api.py

```diff
@@ -1,139 +1,123 @@
+          1.09384397e-04, 0.00000000e+00],
+         [2.32197642e-02, 1.48485349e-02, 1.20572122e-02, 1.00518325e-02,
+          3.78616550e-03, 5.51953330e-04, 1.58034940e-03, 0.00000000e+00,
+          6.80672762e-04, 3.19095445e-03],
+         [3.35171446e-02, 2.30573826e-02, 1.86911281e-02, 1.16252769e-02,
+          6.42230362e-03, 4.63001803e-03, 2.50486028e-03, 1.08768849e-03,
+          0.00000000e+00, 6.75229309e-03],
+         [5.30732870e-02, 3.61515097e-02, 3.07820514e-02, 2.19761301e-02,
+          1.52691351e-02, 9.15373303e-03, 5.41988341e-03, 0.00000000e+00,
+          9.24524793e-04, 3.14691127e-03],
+         [6.05984218e-02, 5.18058091e-02, 4.45022509e-02, 3.48668806e-02,
+          2.37355866e-02, 1.48193939e-02, 1.27216997e-02, 4.51163156e-03,
+          9.03905951e-04, 1.46147527e-03],
+         [6.03375509e-02, 6.00600466e-02, 5.51640466e-02, 4.15391065e-02,
+          3.14524844e-02, 2.64199078e-02, 1.41928447e-02, 6.39168778e-03,
+          5.34856878e-03, 3.28401709e-03],
+         [4.75437082e-02, 5.76229692e-02, 5.73692545e-02, 4.73626107e-02,
+          3.53275351e-02, 2.55322661e-02, 1.92126688e-02, 9.48204752e-03,
+          9.59323533e-03, 3.06630856e-03],
+         [3.94404493e-02, 4.60249558e-02, 4.95812669e-02, 5.13879694e-02,
+          3.92197557e-02, 2.88589876e-02, 1.97741222e-02, 1.41243441e-02,
+          8.03299714e-03, 6.66760467e-03],
+         [3.60199437e-02, 3.75546440e-02, 4.03789952e-02, 4.74576391e-02,
+          4.75049056e-02, 3.51673886e-02, 2.08846033e-02, 1.69556923e-02,
+          1.06233349e-02, 1.02646966e-02]],
+        dtype=np.float32
+    )
+    assert np.allclose(
+        expected_registered_values,
+        registered_image.numpy_array[2, 2, 0, 40:50, 40:50]
     )
-    with warnings.catch_warnings():
-        warnings.simplefilter("ignore")
 
-        # blobs = dots; define the spots in the dots image, but then find them again in the stack.
-        blobs_image = dots.max_proj(Indices.ROUND, Indices.Z)
-        intensities = gsd.run(primary_image, blobs_image=blobs_image)
+    intensities = iss.intensities
 
     # assert that the number of spots detected is 99
     assert intensities.sizes[Features.AXIS] == 99
 
     # decode
-    decoded_intensities = experiment.codebook.decode_per_round_max(intensities)
+    decoded = iss.decoded
 
     # decoding identifies 4 genes, each with 1 count
-    genes, gene_counts = np.unique(decoded_intensities['target'], return_counts=True)
-    assert np.array_equal(genes, np.array(['CD68', 'GUS', 'ST-3', 'VEGF', 'nan']))
-    assert np.array_equal(gene_counts, [1, 1, 1, 1, 95])
-
-    # segment
-    # TODO ambrosejcarr: do these need to be adjusted for the image size?
-    seg = Watershed(
-        dapi_threshold=.16,
-        input_threshold=.22,
-        min_distance=57
-    )
-    regions = seg.run(filtered_primary_image, filtered_nuclei)
+    genes, gene_counts = iss.genes, iss.counts
```

Comment:
I think the way these get calculated, the counts come out sorted. I'm wondering why these are not in sorted order.